services/worker-manager/src/providers/aws.js

```diff
@@ -108,6 +105,7 @@ class AwsProvider extends Provider {
         });
       }
 
+      let spawned;
```

Comment:
[copying over from Slack] I’d have `MaxCount: Math.min(toSpawnCounter, Math.ceil(toSpawnPerConfig / config.capacityPerInstance) )`, on line 115, and the same on line 116. 

Btw I am assuming we want to distribute capacity over configs/regions and not instances?

actually the `toSpawnCounter` also needs division in the `runInstances` call because it counts capacity, not instances. All those variables are capacity except the numbers in `runInstances` call that are instances. I think that might work